test/functional/pdo_sqlsrv/pdo_simple_update_variants.phpt

```diff
@@ -2,150 +2,134 @@
 {
-    try 
-    {
-        $stmt = $conn->exec("CREATE TABLE $tableName ([id] sql_variant, [food] sql_variant, [category] sql_variant)"); 
-    }
-    catch (Exception $e)
-    {
+    try {
+        createTable($conn, $tableName, array("id" => "sql_variant", "food" => "sql_variant", "category" => "sql_variant"));
+    } catch (Exception $e) {
         echo "Failed to create a test table\n";
         echo $e->getMessage();
-    }        
+    }
 }
 
-function InsertData($conn, $tableName, $id, $food, $category)
+function insertData($conn, $tableName, $id, $food, $category)
 {
-    try 
-    {
+    try {
         $query = "INSERT $tableName ([id], [food], [category]) VALUES (:id, :food, :category)";
 
         $stmt = $conn->prepare($query);
         $stmt->bindValue(':id', $id);
         $stmt->bindValue(':food', $food);
         $stmt->bindValue(':category', $category);
-        
-        $result = $stmt->execute();  
-        if ($result)
-            echo "\nAdded $food in $category with ID $id.";  
-    }
-    catch (Exception $e)
-    {
+
+        $result = $stmt->execute();
+        if ($result) {
+            echo "Added $food in $category with ID $id.\n";
+        }
+    } catch (Exception $e) {
         echo "Failed to insert food $food\n";
         echo $e->getMessage();
-    }        
+    }
 }
 
-function UpdateID($conn, $tableName, $id, $food, $category)
+function updateID($conn, $tableName, $id, $food, $category)
 {
     $query = "UPDATE $tableName SET id = ? WHERE food = ? AND category = ?";
     $stmt = $conn->prepare($query);
     $result = $stmt->execute(array($id, $food, $category));
-    
-    if ($result)
-        echo "\nFood $food now updated with new id $id.";
-    else
-        echo "Failed to update ID.\n";        
+
+    if ($result) {
+        echo "Food $food now updated with new id $id.\n";
+    } else {
+        echo "Failed to update ID.\n";
+    }
 }
 
-function UpdateFood($conn, $tableName, $id, $food, $category)
+function updateFood($conn, $tableName, $id, $food, $category)
 {
     $query = "UPDATE $tableName SET food = ? WHERE id = ? AND category = ?";
     $stmt = $conn->prepare($query);
     $result = $stmt->execute(array($food, $id, $category));
-    
-    if ($result)
-        echo "\nCategory $category now updated with $food.";
-    else
-        echo "Failed to update food.\n";     
+
+    if ($result) {
+        echo "Category $category now updated with $food.\n";
+    } else {
+        echo "Failed to update food.\n";
+    }
 }
 
-function FetchRows($conn, $tableName)
+function fetchRows($conn, $tableName)
 {
-    $query = "SELECT * FROM $tableName ORDER BY id";   
-  
-    $stmt = $conn->query($query);  
+    $query = "SELECT * FROM $tableName ORDER BY id";
+    $stmt = $conn->query($query);
 
     $stmt->setFetchMode(PDO::FETCH_CLASS, 'Food');
-    while ($food = $stmt->fetch())
-    {   
-        echo "\nID: " . $food->id . " "; 
+    $foodArray = array();
+    while ($food = $stmt->fetch()) {
```

Comment:
what's the use of $foodArray?

Reply:
it was there before to do the sorting. It's not needed anymore, thx.